src/benchmark/file_async_io_read_micro_benchmark.cpp

```diff
@@ -0,0 +1,174 @@
+#include <fcntl.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
+#include <algorithm>
+#include <iterator>
+#include <numeric>
+#include <random>
+#include <aio.h>
+#include "micro_benchmark_basic_fixture.hpp"
+
+namespace hyrise {
+
+const auto MB = uint32_t{1'000'000};
```

Comment:
as stated in the review of the teaching team, this should be renamded to Mega instead of Megabyte, as 10⁹ represents mega